test/test.cc

```diff
@@ -6512,14 +6512,33 @@ TEST(TaskQueueTest, IncreaseAtomicInteger) {
       new ThreadPool{CPPHTTPLIB_THREAD_POOL_COUNT}};
 
   for (unsigned int i = 0; i < number_of_task; ++i) {
-    task_queue->enqueue(
+    bool queued = task_queue->enqueue(
```

Comment:
Could you use `auto` instead of `bool`?

Reply:
Ok, but doesn't that make the code less readable?

Reply:
It's consistency. I try to use `auto` at all the places where we can apply it.